File: fabric/src/main/java/me/pandamods/pandalib/fabric/PandaLibClientFabric.java
package me.pandamods.pandalib.fabric;

import me.pandamods.pandalib.PandaLib;
import me.pandamods.pandalib.PandaLibClient;
import net.fabricmc.api.ClientModInitializer;
import net.fabricmc.api.EnvType;
import net.fabricmc.api.Environment;
import net.fabricmc.api.ModInitializer;

@Environment(EnvType.CLIENT)
public class PandaLibClientFabric implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        PandaLibClient.init();
    }
}

File: fabric/src/main/kotlin/dev/pandasystems/pandalib/fabric/client/PandaLibClientFabric.kt

package dev.pandasystems.pandalib.fabric.client

import dev.pandasystems.pandalib.client.PandaLibClient
import net.fabricmc.api.ClientModInitializer

class PandaLibClientFabric : ClientModInitializer {
	override fun onInitializeClient() {
		PandaLibClient.init()
	}
}
